### PR TITLE
UI for volunteer registration form - new UI

### PR DESCRIPTION
**Pull request by Satendra-SR**
### Changes - 

- UI changes fixes for Volunteer form

### CODE REVIEW

File: wp-content/themes/goonj-crm/assets/styles/lg.scss [hunk 1]
```diff
@@ -58,6 +58,21 @@
         .error {
             font-size: 14px;
         }
+
+        .af-container:not(:has(.af-container)) {
+            display: flex !important;
+            flex-wrap: wrap !important;
+        }
```
Comment (pokhiii) [inline wp-content/themes/goonj-crm/assets/styles/lg.scss:65]: My understanding is: in our layout we want certain container elements to use a flexbox layout, but only if they do not have nested containers of the same class. This ensures that the outer containers behave as flex containers, while nested containers can have different styling or layout rules applied.

File: wp-content/themes/goonj-crm/assets/styles/lg.scss [hunk 2]
```diff
@@ -58,6 +58,21 @@
...
+    
+        .af-container af-field {
```
Comment (pokhiii) [inline wp-content/themes/goonj-crm/assets/styles/lg.scss:67]: I think `af-field` is a class and hence is missing a dot `.`
```suggestion
        .af-container .af-field {
```

File: wp-content/themes/goonj-crm/assets/styles/lg.scss [hunk 3]
```diff
@@ -37,16 +37,16 @@
...
         
         af-form {
-            width: 520px !important;
             margin-left: auto;
             margin-right: auto;
         }
 
         .select2-search-choice-close {
-            padding-top: 8px !important;
+            padding-top: auto !important;
```
Comment (pokhiii) [inline wp-content/themes/goonj-crm/assets/styles/lg.scss:49]: The value `auto` is not a valid value for the `padding-top` property. See: https://developer.mozilla.org/en-US/docs/Web/CSS/padding-top#syntax

**Review by pokhiii — CHANGES_REQUESTED** [review]

File: wp-content/themes/goonj-crm/assets/styles/lg.scss (hunk 1 above)
Comment (Satendra-SR) [reply wp-content/themes/goonj-crm/assets/styles/lg.scss:61]: Yes, that it correct

File: wp-content/themes/goonj-crm/assets/styles/lg.scss (hunk 2 above)
Comment (Satendra-SR) [reply wp-content/themes/goonj-crm/assets/styles/lg.scss:63]: @pokhiii The **af-field** is a custom tag, not a class in this case.

File: wp-content/themes/goonj-crm/assets/styles/lg.scss (hunk 3 above)
Comment (Satendra-SR) [reply wp-content/themes/goonj-crm/assets/styles/lg.scss]: Corrected this

File: wp-content/themes/goonj-crm/assets/styles/lg.scss (hunk 2 above)
Comment (pokhiii) [reply wp-content/themes/goonj-crm/assets/styles/lg.scss:63]: Ah, my bad.

**Review by pokhiii — APPROVED** [review]

**Pull request merged**